src/pages/Dashboard/Tweet/index.tsx

```
@@ -0,0 +1,67 @@
+      <h1 className="mb-4 text-2xl font-bold">Tweets</h1>
+
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <input {...register('text')} />
+        <button type="submit">Submit</button>
+      </form>
+
+      <div className="mt-4 grid gap-4">
+        {data &&
+          data?.map((tweet: TweetType) => (
+            <TweetCard key={tweet.id} tweet={tweet} />
+          ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)